eth/protocols/eth/peer.go

```diff
@@ -470,6 +470,16 @@ func (p *Peer) RequestTxs(hashes []common.Hash) error {
 	})
 }
 
+// IsTrusted returns whether the peer is a trusted peer or not.
+func (p *Peer) IsTrusted() bool {
+	return p.Info().Network.Trusted
```

Comment:
here and below, for both functions, no risks of `nil` pointers exceptions, right?

Reply:
Yes, I think we should be fine here. Checked the `Info()` function and other occurances of it being called.